src/Microsoft.Extensions.DiagnosticAdapter/Internal/ProxyTypeEmitter.cs

```diff
@@ -167,7 +167,11 @@ private static bool VerifyProxySupport(ProxyBuilderContext context, Tuple<Type,
             var propertyMappings = new List<KeyValuePair<PropertyInfo, PropertyInfo>>();
 
             var sourceProperties = sourceType.GetRuntimeProperties();
-            foreach (var targetProperty in targetType.GetRuntimeProperties())
+            var targetProperties = targetType
+                .GetInterfaces()
+                .SelectMany(i => i.GetRuntimeProperties())
+                .Concat(targetType.GetRuntimeProperties());
+            foreach (var targetProperty in targetProperties)
```

Comment:
There was an undocumented bug here. If your proxy interface inherited from another interface, then the properties on the base interface would all those `MethodMissingException`